frontend/public/style/_vars.scss

```diff
@@ -20,8 +20,18 @@ $fa-font-path: $consoleWebDistPath + '/';
 $font-family-base: var(--pf-global--FontFamily--sans-serif);
 // Patternfly 3 defaults to 12px, which is too small in many cases.
 $font-size-base: 14px;
+// Use PatternFly 4 heading font sizes in Patternfly 3.
+$font-size-h1: 24px; // must be literal value because of calculation in patternfly/dist/sass/patternfly/_blank-slate.scss
```

Comment:
This can be changed after https://github.com/openshift/console/pull/2710 merges.

Reply:
Done.  @spadgett, please re-tag.